client/src/css/background-animation.css

```diff
@@ -0,0 +1,135 @@
+@-webkit-keyframes gradientAnimation {
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+@-moz-keyframes gradientAnimation {
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+@keyframes gradientAnimation { 
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+
+@keyframes moveCircles {
+    0% { transform: translate(-50%, -50%); }
+    50% { transform: translate(50%, 50%); }
+    100% { transform: translate(-50%, -50%); }
+}
+
+@keyframes moveInCircle1 {
+    0% { transform: translate(0, 0); }
+    25% { transform: translate(30px, -50px); }
+    50% { transform: translate(-60px, 30px); }
+    75% { transform: translate(30px, 60px); }
+    100% { transform: translate(0, 0); }
+}
+
+@keyframes moveInCircle2 {
+    0% { transform: translate(0, 0); }
+    25% { transform: translate(-50px, 30px); }
+    50% { transform: translate(60px, -30px); }
+    75% { transform: translate(-30px, -60px); }
+    100% { transform: translate(0, 0); }
+}
+
+@keyframes moveInCircle3 {
+    0% { transform: translate(0, 0); }
+    25% { transform: translate(20px, 40px); }
+    50% { transform: translate(-40px, -20px); }
+    75% { transform: translate(40px, -40px); }
+    100% { transform: translate(0, 0); }
+}
+
+@keyframes moveInCircle4 {
+    0% { transform: translate(0, 0); }
+    25% { transform: translate(-60px, 40px); }
+    50% { transform: translate(40px, -40px); }
+    75% { transform: translate(-40px, 60px); }
+    100% { transform: translate(0, 0); }
+}
+
+.body::before, .body::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 200%;
+    height: 200%;
+    background: radial-gradient(circle, rgba(255, 255, 255, 0.1), transparent);
+    mix-blend-mode: overlay;
+    filter: blur(100px);
+    animation: moveCircles 30s linear infinite;
+}
+  
+.background::after {
+    animation-direction: reverse;
+}
+
+
+.circle-container{
+    max-width: 1920px;
+    width: 100%;
+    height: 100dvh;
+    container: circle-container / size;
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.circle {
+    position: absolute;
+    border-radius: 100%;
+    aspect-ratio: 1 / 1;
+}
+
+.circle1 {
+    width: 28cqmax;
+    top: 20%;
+    left: 12%;
+    animation: moveInCircle1 20s ease-in-out infinite alternate;
+    background: rgb(107,7,217);
+    background: linear-gradient(90deg, rgba(107,7,217,1) 0%, rgba(126,15,142,1) 100%);
+    -webkit-box-shadow:0px 0px 92px 6px rgba(162,0,255,0.35);
+    -moz-box-shadow: 0px 0px 92px 6px rgba(162,0,255,0.35);
+    box-shadow: 0px 0px 92px 6px rgba(162,0,255,0.35);
+}
+
+.circle2 {
+    width: 13cqmax;
+    top: 60%;
+    left: 25%;
+    animation: moveInCircle2 25s ease-in-out infinite alternate-reverse;
+        background: rgb(107,7,217);
```

Comment:
why is `background` here and in the rest of the file to the right of the other columns